journal/journal-app.js

```diff
@@ -106,25 +157,56 @@ function createJournalElement(id, title, documentation, reflection, modalRef) {
     const journalTitle = document.createElement('textarea');
     const journalDocumentation = document.createElement('textarea');
     const journalReflection = document.createElement('textarea');
-    
+
+    // Create the 'Save' button
+    var saveButton = document.createElement('button');
+    saveButton.innerText = 'Save';
+
+    // Apply styles to the 'Save' button
+    saveButton.style.position = 'absolute';
+    saveButton.style.top = '5px';
+    saveButton.style.right = '5px';
+    saveButton.style.fontSize = '12px';
+    saveButton.style.padding = '5px 10px';
+
+    journalBody.style.position = 'relative';
```

Comment:
Again, why do we modify the styling here instead of the CSS file?

Reply:
moved this also to the css file